c/meson.build

```diff
@@ -142,17 +145,40 @@ if get_option('default_library') == 'static'
 endif
 
 
+# Service API
+nns_capi_service_shared_lib = shared_library ('capi-nnstreamer-service',
+  nns_capi_service_srcs,
+  dependencies: nns_capi_deps,
+  include_directories: nns_capi_include,
+  install: true,
+  install_dir: api_install_libdir,
+  version: api_version,
+)
+
+nns_capi_service_static_lib = static_library ('capi-nnstreamer-service',
+  nns_capi_service_srcs,
+  dependencies: nns_capi_deps,
+  include_directories: nns_capi_include,
+  install: true,
+  install_dir: api_install_libdir,
+)
+
+nns_capi_service_dep = declare_dependency(link_with: nns_capi_service_shared_lib)
+if get_option('default_library') == 'static'
+  nns_capi_service_dep = declare_dependency(link_with: nns_capi_service_static_lib)
+endif
+
```

Comment:
How about separating .so file for ml-service from ml-inference?

Reply:
OK!